app/services/analytics_events.rb

```diff
@@ -530,6 +530,12 @@ def idv_personal_key_confirm_visited
     track_event('IdV: personal key confirm visited')
   end
 
+  # @identity.idp.event_name IdV: personal key confirm submitted
+  # User submitted IDV personal key confirmation modal
+  def idv_personal_key_confirm_submitted
+    track_event('IdV: personal key confirm submitted')
```

Comment:
We should add an event key to the list of event keys in Analytics, and maintain this string there.

Reply:
We stopped doing the analytics events constants with the introduction of `AnalyticsEvents` helper methods as of #6014.